sdk/storage/azure-storage-files-shares/test/share_directory_client_test.cpp

```diff
@@ -201,8 +203,10 @@ namespace Azure { namespace Storage { namespace Test {
     Files::Shares::Models::FileShareSmbProperties properties;
     properties.Attributes = Files::Shares::Models::FileAttributes::Directory
         | Files::Shares::Models::FileAttributes::NotContentIndexed;
-    properties.CreationTime = ToIso8601(std::chrono::system_clock::now(), 7);
-    properties.LastWriteTime = ToIso8601(std::chrono::system_clock::now(), 7);
+    properties.CreateOn = Core::DateTime::Parse(
```

Comment:
I think instead of this whole line, you can do Core::DateTime::Now(). You do this in multiple places, I commented only on this instance, but feedback applies to all.